titan-web-client/src/modules/roster/components/BranchCardGrid.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Column from 'titan/components/Grid/Column';
+import Row from 'titan/components/Grid/Row';
+import BranchCard from 'titan/modules/roster/components/BranchCard';
+
+/**
+ * Lists roster member cards in a grid.
+ */
+class BranchCardGrid extends React.Component {
+  constructor (props) {
+    super(props);
+  }
+
+  render () {
+    return (
+      <Row wrap="wrap">
+        {this.props.branches.filter((e) => e.is_enabled === true).map((branch, index) => (
```

Comment:
This expression can be simplified:

- `is_enabled` holds a boolean value.
- in idiomatic JS,  single parameter functions don't need parentheses.

```
this.props.branches.filter(e => e.is_enabled)
```